modules/cudf/src/series.ts

```diff
@@ -344,7 +344,9 @@ export class AbstractSeries<T extends DataType = any> {
    * @param index the index in this Series to set a value for
    * @param value the value to set at `index`
    */
-  setValue(index: number, value: T['scalarType']): void { this.scatter(value, [index]); }
+  setValue(index: number, value: T['scalarType']): Series<T> {
+    return this.scatter(value, [index]);
+  }
```

Comment:
I'll be honest and say the "set" in "setValue" really suggests this is a mutating operation to me, and I think as a user I would be surprised to find otherwise.

Reply:
this operation seems similar to `series` `fill` & `fillInPlace`. does it seem like a good idea to follow the pattern, with `set` and `setInPlace`? or `setValue` & `setValueInPlace`?

Reply:
added

Reply:
As mentioned in https://github.com/rapidsai/node-rapids/pull/159#discussion_r625961144, I tend to agree with @bryevdv here. Even if `setValue()` is implemented via scatter under the hood, the API strongly suggests mutation. IMO it should behave (from the outside) similar to other "array-like" JS types:

```js
const ary = [1, 2, 3];
ary[1] = 4;
```